adodbapi/__init__.py

```diff
@@ -3,30 +3,35 @@
 Copyright (C) 2002 Henrik Ekelund, version 2.1 by Vernon Cole
 * http://sourceforge.net/projects/adodbapi
 """
-import sys
 import time
 
-from .adodbapi import Connection, Cursor, __version__, connect, dateconverter
+from .adodbapi import (
+    Connection as Connection,
```

Comment:
I'm kindof just guessing that these were meant to be re-exported. Otherwise I can remove them instead if they are just unused.
Or leave this as an undecidable case for pycln if they are unused, but don't want to break any code using these re-exports.

Reply:
This seems harmless enough.

Reply:
Yeah explicit re-exports is the safe choice here since it's cosmetic only.

Reply:
IIRC the re-exports were needed to keep from breaking existing code.

Reply:
Sounds good, let's keep it for backwards compatibility reasons
```suggestion
    # Re-exports to keep backward compatibility with existing code
    Connection as Connection,
```